config/jobs/kubernetes/sig-cloud-provider/aws/kops/kops-periodics.yaml

```diff
@@ -235,15 +235,15 @@ periodics:
       - --cluster=e2e-kops-aws-containerd.test-cncf-aws.k8s.io
       - --deployment=kops
       - --env=KUBE_SSH_USER=ec2-user
-      - --extract=ci/latest
+      - --extract=release/stable-1.17
```

Comment:
any reason for using stable-1.17 here rather than all of the other e2e periodics using latest ?

Reply:
I checked the latest failed tests for containerd. Seems it cannot download the images for dev versions. I either the images are pre-loaded using some docker commands or need some credentials that containerd cannot read.
I would have to dig and test a bit on how to get those, and would mean a lot of trial and error. I'd rather wait until I don't need so many "ok-to-test".
Also, for Kops is not really that important to run on latest dev.

Reply:
I can also remove that part for now and dig a bit more.

Reply:
ok, thats a reasonable compromise. We can get that working on latest later.